tests/mock_resources.py

```diff
@@ -20,7 +20,7 @@ class MySubResource(Resource):
 
 
 class Error(Resource):
-    schema = {"object": str, "message": str, "params": list, "type": str}
```

Comment:
This was incorrectly adding `object` to the test framework's definition of the `Error` class. This prevented the tests from finding the issue being fixed.